refinery/data_set_manager/utils.py

```diff
@@ -615,12 +613,8 @@ def _generate_solr_params(params, assay_uuids=[]):
                   'facet.limit=-1'
                   ])
 
-    solr_params = 'fq=({})'.format(
-        ' OR '.join(map(
-            lambda id: 'assay_uuid:{}'.format(id),
-            assay_uuids
-        ))
-    )
+    assert len(assay_uuids) > 0, 'At least one assay must be accessible'
```

Comment:
I don't think `assert` is inline with what we've been doing. We generally have been using try/excepts, but i do enjoy its simplicity. Maybe it could be up for debate?

Presumably 
AssertionError is raised? so at the very least that should be handled.